Makefile

```diff
@@ -308,9 +308,10 @@ talosctl-cni-bundle: ## Creates a compressed tarball that includes CNI bundle fo
 .PHONY: cloud-images
 cloud-images: ## Uploads cloud images (AMIs, etc.) to the cloud registry.
 	@docker run --rm -v $(PWD):/src -w /src \
-		-e TAG=$(TAG) -e ARTIFACTS=$(ARTIFACTS) \
+		-e TAG=$(TAG) -e ARTIFACTS=$(ARTIFACTS) -e ABBREV_TAG=$(ABBREV_TAG) \
```

Comment:
Added ABBREV_TAG to environment variable here and in cloud-image-uploader.sh passed as an argument for the tag to be compared without the build metadata.